<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="handleAV">
    <link name="handle">
      <inertial>
        <pose>0.0265 0.0005 0.0448 0 0 0</pose>
        <mass>0.05</mass>
        <inertia>
          <ixx>1.02e-06</ixx>
          <ixy>8.73e-09</ixy>
          <iyy>2.61e-06</iyy>
          <ixz>-5.03e-07</ixz>
          <iyz>8.02e-09</iyz>
          <izz>2.11e-06</izz>
        </inertia>
      </inertial>
      <collision name="collision">
        <geometry>
          <mesh>
            <uri>model://handleAV/meshes/door_handle.dae</uri>
          </mesh>
        </geometry>
        <surface>
          <contact>
            <ode>
              <max_vel>0.1</max_vel>
              <min_depth>0.001</min_depth>
            </ode>
          </contact>
          <friction>
            <ode>
              <mu>1.0</mu>
              <mu2>1.0</mu2>
            </ode>
          </friction>
        </surface>
      </collision>
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://handleAV/meshes/door_handle.dae</uri>
          </mesh>
        </geometry>
        <material>
          <ambient>1 0 0 1</ambient>
          <diffuse>1 0 0 1</diffuse>
          <specular>0.1 0.1 0.1 1</specular>
          <emissive>0 0 0 0</emissive>
        </material>
      </visual>
    </link>
  </model>
</sdf>
